tests/Microsoft.VisualStudio.Editors.UnitTests/Microsoft.VisualStudio.Editors.UnitTests.vbproj

```diff
@@ -0,0 +1,40 @@
+﻿<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information. -->
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="..\..\build\import\VisualStudio.props" />
+  <PropertyGroup>
+    <RootNamespace>Microsoft.VisualStudio</RootNamespace>
+  </PropertyGroup>
+  <ItemGroup>
+    <Compile Remove="My Project\**" />
+    <EmbeddedResource Remove="My Project\**" />
+    <None Remove="My Project\**" />
+  </ItemGroup>
+  <ItemGroup>
+    <EmbeddedResource Remove="Resource1.resx" />
+  </ItemGroup>
+  <ItemGroup>
+    <None Remove="explorer.ico" />
+    <None Remove="img0.jpg" />
+    <None Remove="img105.jpg" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="explorer.ico">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+    <Content Include="img105.jpg">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </Content>
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\DeployTestDependencies\DeployTestDependencies.csproj" ReferenceOutputAssembly="false" />
+    <ProjectReference Include="..\..\src\Microsoft.VisualStudio.Editors\Microsoft.VisualStudio.Editors.vbproj" />
+    <ProjectReference Include="..\..\src\Microsoft.VisualStudio.AppDesigner\Microsoft.VisualStudio.AppDesigner.vbproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Update="Resource1.Designer.vb">
```

Comment:
Is this compile item necessary? Look like there are various files mentioned in here that aren't being added. Consider cleaning them up.

Reply:
Removed these statements.